### PR TITLE
refactor(confirmdialog): connect it

### PR DESCRIPTION
**Pull request by jmfrancois**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

the confirm dialog declare it's own state through redux

**What is the new behavior?**

the confirm dialog use cmfConnect

**Does this PR introduce a breaking change?**

- [x] No

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://288.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://288.talend.surge.sh)

File: packages/containers/src/ConfirmDialog/ConfirmDialog.test.js [hunk 1]
```diff
@@ -4,10 +4,7 @@ import { fromJS, Map } from 'immutable';
 import { store, Provider } from 'react-cmf/lib/mock';
 
 import Container, { DEFAULT_STATE } from './ConfirmDialog.container';
```
Comment (sdwvit) [inline packages/containers/src/ConfirmDialog/ConfirmDialog.test.js]: eslint: DEFAULT_STATE is not used here, pls check

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://288.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://288.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://288.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://288.talend.surge.sh)

**Pull request merged**